src/main/resources/templates/fragments/exam-number-input.html

```diff
@@ -4,8 +4,8 @@
 <div>
     <form action="#" th:action="@{/search}" th:object="${examNumberForm}" method="post">
         <label for="examNumber">Prüfungsnummer eingeben*:</label>
-        <input type="text" name="examNumber" id="examNumber"
-               placeholder="20202" th:value="*{examNumber}" required>
+        <input th:type="text" name="examNumber" id="examNumber" oninput="this.value = this.value.replace(/\D+/g, '')"
```

Comment:
What is this 'th:' for here?

Reply:
removed it since it did not change anything